packages/faustwp-core/tests/components/WordPressTemplate.test.tsx

```diff
@@ -27,56 +27,6 @@ describe('<WordPressTemplate />', () => {
     );
   });
 
-  test('Properly determines whether or not the given URL is a preview or not', async () => {
```

Comment:
Problematic unit test with lots of testing anti-patterns. I removed this since we already test the logic of isWordPressPreview